packages/sdk/client-services/src/packlets/services/service-host.ts

```diff
@@ -243,7 +244,36 @@ export class ClientServicesHost {
       SystemService: this._systemService,
 
       IdentityService: new IdentityServiceImpl(
-        (params) => this._serviceContext.createIdentity(params),
+        async (params) => {
+          const identity = await this._serviceContext.createIdentity(params);
+
+          // Setup default space.
+          await this._serviceContext.initialized.wait();
+          const space = await this._serviceContext.dataSpaceManager!.createSpace();
+          const obj: TypedObject = new Properties();
+          obj[defaultKey] = identity.identityKey.toHex();
+          const snapshot = obj[base]._createSnapshot();
+          console.log({ obj, snapshot });
+          await this._serviceRegistry.services.DataService!.write({
+            spaceKey: space.key,
+            batch: {
+              objects: [
+                {
+                  objectId: obj[base]._id,
+                  genesis: {
+                    modelType: obj[base]._modelConstructor.meta.type,
```

Comment:
perhaps extract this to a helper function: `createGenesisMutationFromTypedObject`